gdk/gdk.device.lisp

```diff
@@ -1491,6 +1558,26 @@ get_device_settings (GdkDevice *device)
 ;;; Since 3.0
 ;;; ----------------------------------------------------------------------------
 
+(defcfun ("gdk_device_get_axis_value" %gdk-device-get-axis-value) :boolean
+  (device (g-object gdk-device))
+  (axes (:pointer :double))
+  (axis-label gdk-atom-as-string)
+  (value (:pointer :double)))
+
+(defun gdk-device-get-axis-value (device axes axis-label)
+  (assert (= (gdk-device-n-axes device) (length axes)))
+  (with-foreign-objects ((axes-ar :double (gdk-device-n-axes device))
+                         (value :double))
+    (let ((i 0))
```

Comment:
Do you mind using `iterate` here to be a bit more consistent?

Reply:
I'm not familiar with `iterate`, but will do.

Also, same remark applies as for the `assert` test (code copied from `gdk-device-get-axis`).

Reply:
BTW, the point of using `map` is that it supports all sequence types. AFAIK `loop` cannot do that. Apparently `iterate` has an `in-sequence` operator that does it.

Reply:
Yup. I'd go for `in-sequence` as well.